Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt

```diff
@@ -174,6 +174,7 @@ private fun TopAppBarTitle(
     )
 }
 
+@Suppress("DEPRECATION") // ConstraintLayout
```

Comment:
Is the new constraintlayout artifact not working for you? In case, put a note that this is temporary